admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/UISpecServiceImpl.java

```diff
@@ -149,14 +155,19 @@ public UISpecResponseDto defineUISpec(UISpecDto dto) {
 		uiSpecEntity.setId(UUID.randomUUID().toString());
 		uiSpecEntity.setIsDeleted(false);
 		uiSpecEntity.setEffectiveFrom(LocalDateTime.now(ZoneId.of(ZoneOffset.UTC.getId())));
+		List<UISpec> uispeclist = uiSpecRepository.findUISpecbyDomainandType(dto.getDomain(), dto.getType());
```

Comment:
Follow method naming convention findUISpecByDomainAndType

Reply:
Updated the name as per your comment